package.json

```diff
@@ -4,6 +4,10 @@
   "description": "A GitHub action for uploading OAS files to ReadMe Micro",
   "main": "index.js",
   "scripts": {
+    "pretest": "npm run lint && npm run prettier",
```

Comment:
this feels like it should be its own command to run in isolation of the tests. I suppose you could run 'npm run pretest'.